packages/files-ui/src/Components/Modules/Settings/index.tsx

```diff
@@ -171,7 +163,7 @@ const Settings: React.FC = () => {
         <div className={classes.tabsContainer}>
           <Tabs
             activeKey={path}
-            onTabSelect={onSelectTab}
+            onTabSelect={onSelectTab as (key: string) => void}
```

Comment:
The typing was all right afaik
```suggestion
            onTabSelect={onSelectTab}
```

Reply:
Was throwing for me

Reply:
It should not anymore now that you applied my previous changes, with the undefined changed to empty string :)